vllm/engine/arg_utils.py

```diff
@@ -1225,19 +1225,24 @@ def _override_v1_engine_args(self, usage_context: UsageContext) -> None:
         """
         assert envs.VLLM_USE_V1, "V1 is not enabled"
 
+        # V1 always uses chunked prefills.
+        self.enable_chunked_prefill = True
```

Comment:
Can we throw an overriding warning if user sets to False?

Reply:
I see then it should be fine as we already have a log saying chunked prefill is enabled IIRC.